libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h

```diff
@@ -38,6 +38,7 @@ class ESP8266WiFiSTAClass {
 
         wl_status_t begin(const char* ssid, const char *passphrase = NULL, int32_t channel = 0, const uint8_t* bssid = NULL, bool connect = true);
         wl_status_t begin(char* ssid, char *passphrase = NULL, int32_t channel = 0, const uint8_t* bssid = NULL, bool connect = true);
+        wl_status_t begin(const String& ssid, const String& passphrase = String('\0'), int32_t channel = 0, const uint8_t* bssid = NULL, bool connect = true);

```

Comment:
`emptyString` can be used instead of `String('\0')` or `String()` (recently added in WString.h for that purpose)